pkg/controller/directvolumemigration/migrate.go

```diff
@@ -10,6 +11,32 @@ import (
 )
 
 func (r *ReconcileDirectVolumeMigration) migrate(direct *migapi.DirectVolumeMigration) (time.Duration, error) {
+
+	// Ready
+	migration, err := direct.GetMigrationForDVM(r)
+	if err != nil {
+		return 0, liberr.Wrap(err)
+	}
+	if migration == nil {
+		log.Info("Migration not found for DVM", "name", direct.Name)
+		return 0, liberr.Wrap(err)
```

Comment:
I think @dymurray mentioned that the DVM should be allowed to run even if there is no associated MigMigration, right? Sometimes you guys do separate DVM operators for testing/debugging purposes. 

It looks like the way this is written, lack of a linked migmigration will cause the DVM to stop.

This should be written such that it still work when a user creates a DVM separate from a migmigration.

Reply:
Correct, we still want to support running DVM outside of the context of migmigration controller

Reply:
Update the PR.